docs/migration-2-to-3.md

```diff
@@ -31,3 +32,53 @@ dependency management of Spring Boot as mentioned in [the migration guide](https
 
 Please note that Wiremock repackaged `javax` classes.
 They will be available in tests but should not be used in your code.
+
+
+## Tracing
+
+Tracing uses OpenTelemetry now.
+Any configuration that references `zipkin` or `sleuth` is not considered anymore.
+
+Please refer to the respective `management.tracing` properties in
+[the documentation](starter-web.md#configuration).
+
+
+## AsyncAPI generation
+
+Json Schemas for AsyncAPI are generated with
+[Victools' Json Schema Generator](https://github.com/victools/jsonschema-generator) now.
+The [previously used library](https://github.com/mbknor/mbknor-jackson-jsonSchema) is barely
+maintained in the past years.
+
+The old library provided their own annotations.
+Now, annotations of Jackson (e.g. `@JsonSchemaDescription`), Swagger (e.g. `@Schema`) and Jakarta
+Validation (e.g. `NotNull`) can be used.
+Note that not all attributes of all annotations are covered and multiple examples are not possible
+anymore.
+Only one example can be defined with `@Schema(example = "value")`.
+
+Discovery of Schema root classes and therefore the Java API of the `AsyncApiGenrator` has changed.
```

Comment:
This sentence somehow doesn't feel finished!

Reply:
I still feel that the sentence can be improved! I will think about how I can re-write it and provide a suggestion later :)